idseq_dag/steps/run_lzw.py

```diff
@@ -1,8 +1,126 @@
 from idseq_dag.engine.pipeline_step import PipelineStep
 import idseq_dag.util.command as command
-
+import idseq_dag.util.log as log
 
 class PipelineStepRunLZW(PipelineStep):
+
     def run(self):
-        for f in self.output_files_local():
-            command.execute("date > %s" % f)
+        input_fas = self.input_files_local[0]
+        output_fas = self.output_files_local()
+        cutoff_fractions = self.additional_attributes["thresholds"]
+        PipelineStepRunLZW.generate_lzw_filtered(input_fas, output_fas, cutoff_fractions)
```

Comment:
would you rather 'PipelineStepRunLZW' be 'self'?

Reply:
yeah. That's what the linter told me ;-)